subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidBuildPerformanceTest.groovy

```diff
@@ -50,6 +52,11 @@ class RealLifeAndroidBuildPerformanceTest extends AbstractCrossVersionGradleProf
         runner.runs = runs
         applyEnterprisePlugin()
 
+        and:
+        if (testProject == SANTA_TRACKER_KOTLIN) {
+            testProject.configureForLatestAgpVersionOfMinor(runner, SANTA_AGP_VERSION_MINOR)
```

Comment:
Why not configure it for the `perf-android-large` as well?

Reply:
`perf-android-large` is using AGP 3.5.
I'd like to keep that change set focused on Santa Tracker.